Agrume/Agrume.swift

```diff
@@ -217,6 +220,26 @@ public final class Agrume: UIViewController {
     }
     return _spinner!
   }
+	
+	fileprivate var _closeBtn: UIButton?
+	fileprivate var closeBtn: UIButton {
+		if _closeBtn == nil {
+			let closeBtn = UIButton(type: .system)
+			closeBtn.setImage(UIImage(named: "close_btn"), for: UIControlState.normal)
+			closeBtn.tintColor = backgroundColor == UIColor.black ? UIColor.black : UIColor.white
+			closeBtn.addTarget(self, action: #selector(dismissAfterFlick), for: .touchUpInside)
+			closeBtn.translatesAutoresizingMaskIntoConstraints = false
+			view.addSubview(closeBtn)
+			let horizontalConstraint = NSLayoutConstraint(item: closeBtn, attribute: NSLayoutAttribute.trailing, relatedBy: NSLayoutRelation.equal, toItem: view, attribute: NSLayoutAttribute.trailing, multiplier: 1, constant:0)
+			let verticalConstraint = NSLayoutConstraint(item: closeBtn, attribute: NSLayoutAttribute.top, relatedBy: NSLayoutRelation.equal, toItem: view, attribute: NSLayoutAttribute.top, multiplier: 1, constant: 20)
```

Comment:
I can see why you added the width and height constraints with the random multiplier but the good way is to use the intrinsic sizing of the button instead. Again, if it's a button passed in rather than "hardcoded" here, that won't be necessary.